lib/sorbet-rails/model_rbi_formatter.rb

```diff
@@ -16,14 +16,14 @@ class ModelRbiFormatter
   def initialize(model_class, available_classes)
     @model_class = model_class
     @available_classes = available_classes
-    @columns_hash = model_class.columns_hash
+    @columns_hash = model_class.table_exists? ? model_class.columns_hash : {}
     @generated_sigs = ActiveSupport::HashWithIndifferentAccess.new
     @generated_class_sigs = ActiveSupport::HashWithIndifferentAccess.new
     @generated_scope_sigs = ActiveSupport::HashWithIndifferentAccess.new
     @generated_querying_sigs = ActiveSupport::HashWithIndifferentAccess.new
     begin
       # Load all dynamic instance methods of this model by instantiating a fake model
-      @model_class.new
+      @model_class.new unless @model_class.abstract_class?
     rescue StandardError
```

Comment:
Also: TIL that `NotImplementedError` does not inherit from `StandardError`.

Reply:
This is a good addition, but does this mean we still generate an (almost) empty file for this, or there can be an expected rbi file we can add to the test?

Reply:
>  does this mean we still generate an (almost) empty file for this

Yeah, currently it'd produce an `application_record.rbi` file, and it'll contain all the generated stuff we'd normally see in a model rbi. (note that the rake task filters it out, so we wouldn't hit this line from the rake task anyway.)

I was a little torn on what to do about this case. But if a user asks to generate rbis for an abstract class, it seems to me that we should do it! 

I'd vote to add an expected rbi for this case.

Reply:
OK, cool—I just added a new abstract class `Potion`, and an expected output rbi for it (just generates relations, no column accessors.)

This seemed like it'd be less confusing than having an `expected_application_record.rbi`, because in the real world people will use the rake task, and that should be blacklisting `ApplicationRecord`.

Reply:
I agree. Thanks for adding the test!